core/shared/src/main/scala/fs2/Stream.scala

```diff
@@ -1417,6 +1417,48 @@ object Stream {
     def changes(implicit eq: Eq[O]): Stream[F,O] = self.filterWithPrevious(eq.neqv)
 
     /**
+     * Runs the supplied stream in the background as elements from this stream are pulled.
+     *
+     * The resulting stream terminates upon termination of this stream. The background stream will
+     * be interrupted at that point. Early termination of `that` does not terminate the resulting stream.
+     *
+     * Any errors that occur in either `this` or `that` stream result in the overall stream terminating
+     * with an error.
+     *
+     * Upon finalization, the resulting stream will interrupt the background stream and wait for it to be
+     * finalized.
+     *
+     * This method is similar to `this mergeHaltL that.drain` but ensures the `that.drain` stream continues
```

Comment:
I like this explanation, but it refers to `mergeHaltL`, which is removed. Likely to be confusing to someone who's not familiar with the old API perhaps?